content/blog/noetic-eol/index.md

```diff
@@ -0,0 +1,166 @@
+In the proceeding five years we focused our efforts on making ROS 2 a world class framework for professional robot development, and our user community seems to agree.
+According to September 2024 download data from our servers ROS 2 downloads now make up almost 80% of all ROS downloads.
+We've now seen ROS 2 used successfully in production by a large number of companies, across dozens of industries, and we think it is time for the last remaining ROS users to make the switch. 
+
+If you have been directed to this page it was probably because you saw one of the notifications we slipped into the source code for ROS Noetic or RViz. 
+We've heard from a number of users that they were unaware that ROS 1 Noetic, and consequently all of 1, will go end-of-life on **May 31st, 2025**.
+We're sorry that we interrupted your work, but it is crucial that you know what happens once ROS 1 goes end-of-life.
+
+<img src="/imgs/distro-download-data.png" alt="ROS Downloads by Distro, September 2024" style="width:600px;align:center"/>
+
+### What Does End-of-Life Mean?
+
+Users often ask us, “what does end of life mean?” 
+To put it briefly, end of life means that the ROS team will no longer support that particular ROS release. 
+In practical terms, this means that after May 31st of 2025 we will no longer be providing the following for any ROS 1 release, including Noetic:
+
+* New features or capabilities
+* Security updates
+* Bug fixes, patches or support
+* Updated binaries
+
+After May 31st, 2025 ROS 1 Noetic may find themselves exposed to un-patched security vulnerabilities and bugs, with few avenues of support short of fixing these issues themselves and recompiling Noetic from source. 
+
+It is also worth *noting the things that won’t change after ROS 1* goes end of life.
+
+* ROS 1 binaries will not suddenly disappear from our servers. They will continue to be hosted in the archive in perpetuity. 
+* ROS 1 projects will not suddenly stop working on May 31st. 
+* Users do not need to migrate immediately, but they should have migration plan and upgrade as soon as is reasonable
+
+
+<br>
+
+### ROS 2 Migration Options
+
+**If you are still using ROS 1 it is time to consider switching to ROS 2 Humble Hawksbill ([migration guide](https://docs.ros.org/en/humble/How-To-Guides/Migrating-from-ROS1.html)) or ROS 2 Jazzy Jalisco ([migration guide](https://docs.ros.org/en/jazzy/How-To-Guides/Migrating-from-ROS1.html)).**
+Which distro you choose as your migration target  is going to depend on the complexity of your ROS 1 system.
+This [migration heuristic tool](https://github.com/osrf/ros2-porting-heuristics) can provide you with estimate the complexity of  migrating your project to ROS 2.
+If have a you smaller ROS 1 project, or a reasonably de-coupled workspace, it is most likely that you can migrate ROS 2 Jazzy Jalisco directly.
```

Comment:
I would recommend putting this info into the ROS 1 -> 2 migration guide, and linking that from here. https://docs.ros.org/en/rolling/How-To-Guides/Migrating-from-ROS1.html